modules/nodes-cluster-resource-levels-job.adoc

```diff
@@ -53,6 +53,29 @@ EOF
 ----
 $ oc create sa cluster-capacity-sa
 ----
++
+[TIP]
+====
+You can alternatively apply the following YAML to create the service account:
+
+.Sample `cluster-capacity-sa` service account
+[source,yaml]
+----
+apiVersion: v1
+imagePullSecrets:
+- name: cluster-capacity-sa-dockercfg-gbgmn
+kind: ServiceAccount
+metadata:
+  creationTimestamp: "2021-05-13T20:10:52Z"
+  name: cluster-capacity-sa
+  namespace: default
+  resourceVersion: "43851"
+  uid: 8c7fd1ec-7219-4a86-b7fd-96ca121b13a9
```

Comment:
@mikemckiernan Why do we leave the `creationTimestamp`, `resourceVersion` and `uid` in this example, but you explicitly removed them in https://github.com/openshift/openshift-docs/pull/32472#discussion_r656556805

Reply:
I removed the TIPs from this file because we don't have information about how the `oc adm` command creates the secrets that are associated with the service account.  Let's touch base to see how we can identify an SME for information.